cmd/remote.go

```diff
@@ -62,22 +62,48 @@ inerta remote status gcloud`,
 
 // addCmd represents the remote add command
 var addCmd = &cobra.Command{
-	Use:   "add",
+	Use:   "add [REMOTE]",
 	Short: "Add a reference to a remote VPS instance",
 	Long: `Add a reference to a remote VPS instance. Requires 
 information about the VPS including IP address, user and a PEM
-file. Specify a VPS name and an IP address.`,
-	Args: cobra.MinimumNArgs(2),
+file. Specify a VPS name.`,
+	Args: cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		// Ensure project initialized.
 		_, err := client.GetProjectConfigFromDisk()
 		if err != nil {
 			log.WithError(err)
 		}
-		user, _ := cmd.Flags().GetString("user")
-		pemLoc, _ := cmd.Flags().GetString("identity")
+
 		port, _ := cmd.Flags().GetString("port")
-		err = client.AddNewRemote(args[0], args[1], user, pemLoc, port)
+
+		homeEnvVar := os.Getenv("HOME")
+		sshDir := filepath.Join(homeEnvVar, ".ssh")
+		defaultSSHLoc := filepath.Join(sshDir, "id_rsa")
+
+		var response string
+		fmt.Println("Enter location of PEM file (leave blank to use '" + defaultSSHLoc + "'): ")
+		_, err = fmt.Scanln(&response)
+		if err != nil {
+			response = defaultSSHLoc
+		}
+		pemLoc := response
+
+		fmt.Printf("Enter IP address of remote: \n")
+		_, err = fmt.Scanln(&response)
+		if err != nil {
+			log.Fatal("That is not a valid IP address - please try again.")
```

Comment:
I don't think this is an accurate error message. This error means `Scanln` failed, not that the input wasn't a valid IP

Reply:
An error gets thrown if don't input anything as well (hit 'enter' straight away) - plus anything that causes `Scanln` to fail would be an invalid input right? 😅